source/common/access_log/grpc_access_log_impl.cc

```diff
@@ -132,6 +132,13 @@ void HttpGrpcAccessLog::responseFlagsToAccessLogResponseFlags(
   if (request_info.getResponseFlag(RequestInfo::ResponseFlag::RateLimited)) {
     common_access_log.mutable_response_flags()->set_rate_limited(true);
   }
+
+  // @saumoh: TBD To the accesslog.proto
+  //
+  // if (request_info.getResponseFlag(RequestInfo::ResponseFlag::Unauthorized)) {
+  //  common_access_log.mutable_response_flags()->set_unauthorized(true);
+  // }
```

Comment:
?

Reply:
We need to add another flag for `unauthorized` to [accesslog.proto](https://github.com/envoyproxy/data-plane-api/blob/b4a303154703d19f49b63e5529877772ce5da617/api/filter/accesslog/accesslog.proto#L116) (I think)

Will remove it here and add it once the change in protobuf is there. How do we track this kind of TBD? Thx.

Reply:
`TODO(saumoh): the thing TBD`